esmvaltool/recipes/ref/recipe_tcre.yml

```diff
@@ -0,0 +1,95 @@
+# ESMValTool
```

Comment:
Why are there two almost identical copies of the recipe? I would recommend adding just one copy to avoid future synchronization issues.

Reply:
My idea was to have a collection of all recipes used in the REF in one place, but I agree that this will lead to duplication. I can remove it.

Reply:
Done in https://github.com/ESMValGroup/ESMValTool/pull/3904/commits/fbf631153194d7160de5ee0c3b2625d89114b520, but I added the recipe to the doc in https://github.com/ESMValGroup/ESMValTool/pull/3904/commits/3a0bcffd2cd0bc1d5e4e4cde552c0074fbc89ab9